db/migrate/20240917010803_create_evaluation_forms.rb

```diff
@@ -0,0 +1,19 @@
+class CreateEvaluationForms < ActiveRecord::Migration[7.2]
+  def change
+    create_table :evaluation_forms do |t|
+      t.string :title, null: false
+      t.string :instructions, null: false
+      t.integer :challenge_phase, null: false
```

Comment:
we can leave off the `null: false` from these fields to allow them to be empty in the DB